vignettes/zip_zap_models.Rmd

```diff
@@ -0,0 +1,185 @@
+---
+title: "Devel: ZIP and ZAP models"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Devel: ZIP and ZAP models}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+header-includes:
+  - \newcommand{\bm}[1]{\boldsymbol{#1}}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+(Vignette under construction!)
+
+```{r setup, eval=TRUE, echo=TRUE, include=TRUE}
+library(tidyverse)
```

Comment:
Do not import the entire tidyverse here; only load the specific packages you need; likely `dplyr` and `magrittr`. Every package loaded by a vignette must be listed in Suggests (if the package doesn't already depend on it), and I don't want to force a complete tidyverse dependency. inlabru already depends on `dplyr` and `magrittr`; see DESCRIPTION.